src/fges.jl

```diff
@@ -275,33 +323,36 @@ function findBestDelete(step, dataParsed, g, x, y)
     # NAxy: any nodes that are undirected neighbors of y and connected to x by any edge
     # Hyx: any subset of the undirected neighbors of y that are connected to x
     NAyx = calcNAyx(g, y, x)
+    Na = adj_neighbors(g, x, y)
     Hyx = NAyx
+    @assert sort(NAyx) == Na
 
-    # Ceate two containers to hold the best found score and best subset of Tyx
-    bestScore = zero(step.Δscore)
+    # Best found score difference
+    # and best subset of Hyx
+    bestΔ = zero(step.Δscore)
     bestH = Vector{Int}()
 
     # Loop through all possible subsets of Hyx
     for H in powerset(Hyx)
         # Calculate NAyx \ {H}
-        NAyx_H = setdiff(NAyx,H)
+        NAyx_H = setdiff(NAyx, H)
 
         # Check if the operator is valid
         if isclique(g, NAyx_H)
 
             # Score the valid operator 
-            PAy = parents(g,y)
+            PAy = inneighbors(g, y)
```

Comment:
Chickering is not clear about this but I think this is how it should be in our implementation of a PDAG.

Reply:
Proposition 31 suggest an interpretation of `Pa` in PDAGs
<img width="1136" alt="Screenshot 2023-07-23 at 20 14 51" src="https://github.com/mschauer/CausalInference.jl/assets/1923437/876b9c09-b777-4a0b-bdc8-130f1ee908fb">